### ISSUE TITLE
[Question] Apply transform matrix to stage

### ISSUE THREAD

**Issue by JonasGruenwald** [michalochman/react-pixi-fiber#66]
Hi,

I was just wondering if there is a way to apply a transformation matrix to the entire stage, or a specific collection.

I have a couple of components, similar to the `Rectangle` in the example, and I have a regular transformation matrix that I would like to apply to all of them, is there any way to do this?

Thanks for your work on this library!

**Comment by michalochman** [comment]
Hey @JonasGruenwald, you should be able to apply a transformation matrix to any Container using [`updateTransform`](http://pixijs.download/dev/docs/PIXI.Container.html#updateTransform) method. You can do it with React `ref` API.

**Comment by JonasGruenwald** [comment]
Thanks for your reply Michael,

for some reason `updateTransform` didn't do anything for me, although I made sure to convert the matrix to a valid PIXI.Transform object first.

I resorted to just setting the props for `x`, `y` and `scale` on the container for now, since I don't really need rotation or skew and I have the matrix as a prop as well, so I won't have to use the ref on every prop change or something.

Thanks for your help anyway!
